vector/src/main/java/im/vector/adapters/VectorParticipantsAdapter.java

```diff
@@ -816,6 +817,9 @@ public void onSuccess(Void info) {
             matrixUserBadge.setVisibility(View.GONE);
         }
 
+        // Add alpha if cannot be invited
+        convertView.setAlpha(participant.mIsValid ? 1f : 0.5f);
+
```

Comment:
convertView.setEnabled(participant.mIsValid)  should avoid triggering the setOnChildClickListener.onclick event.